src/irmin-pack/pack.ml

```diff
@@ -269,10 +272,12 @@ struct
           else Tbl.add t.staging k v;
           Lru.add t.lru k v
 
+    let return = Lwt.pause
```

Comment:
I renamed it to `pause` and hopefully it's more explicit this way. But I would like to keep this function, because we might have to replace with `Lwt_main.yield` or `auto_yield` if the freeze thread blocks again, and its easier to do it this way.

Reply:
Sure, sounds good. Note that both `yield` and `auto_yield` are unix-only, so we probably cannot rely on them long-term.